modules/caddyhttp/opentelemetry/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenTelemetry module
+
+This module provides integration with OpenTelemetry tracing facilities. It is implemented
+as `caddyhttp.MiddlewareHandler` and can be chained into a list of other handlers.
+
+When enabled, it propagates an existing tracing context or will init a new one otherwise.
+
+It is based on `https://github.com/open-telemetry/opentelemetry-go`.
+
+## Configuration
+
+### Environment variables
+
+It can be configured using environment variables defined
+by [spec](https://github.com/open-telemetry/opentelemetry-specification).
+
+IMPORTANT: Please, consider the version
+of [https://github.com/open-telemetry/opentelemetry-go](https://github.com/open-telemetry/opentelemetry-go). Some parts
+of the specification may be not implemented yet.
+
+Required by module environment variables are: OTEL_EXPORTER_OTLP_PROTOCOL/OTEL_EXPORTER_OTLP_TRACES_PROTOCOL and
+OTEL_PROPAGATORS.
```

Comment:
How about:

```suggestion
These environment variables are required by this module:
- `OTEL_EXPORTER_OTLP_PROTOCOL` or `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL`
- `OTEL_PROPAGATORS`
```

(I'm not sure exactly what you mean though - maybe adjust this)

Reply:
You understood it absolutely correctly 😃 . And I think the proposed by you change sounds clearer. 👍